include/aspect/initial_conditions/interface.h

```diff
@@ -94,7 +94,6 @@ namespace aspect
 
 
 
-
```

Comment:
Please undo this line deletion (and the single line addition in `interface.cc`). Such unrelated changes make it unnecessarily difficult to merge multiple pending patches. I suspect that it wouldn't matter in the current case, but as general good practice, we like to keep changes in each pull request as minimal and self contained as possible.

If you feel that the line really should go for other reasons (e.g., because we typically use 3 empty lines between class and function definitions), then this can be done in a separate pull request devoted to this one task.